subsys/bluetooth/mesh/net.c

```diff
@@ -499,7 +499,7 @@ int bt_mesh_net_send(struct bt_mesh_net_tx *tx, struct net_buf *buf,
 
 	/* Deliver to local network interface if necessary */
 	if (bt_mesh_fixed_group_match(tx->ctx->addr) ||
-	    bt_mesh_elem_find(tx->ctx->addr)) {
+	    bt_mesh_has_addr(tx->ctx->addr, false)) {
```

Comment:
IMO, we should be forwarding the loopback packets as well. We can ignore them in the callback if required.

Reply:
Sure thing, I don't see any problem with that.